install_config/monitoring/modules/enabling_persistent_storage.adoc

```diff
@@ -0,0 +1,14 @@
+= Enabling persistent storage
+
+By default, persistent storage is disabled for both Prometheus time-series data and for Alertmanager notifications and silences. You can configure the cluster to persistently store any one of them or both.
+
+.Procedure
+
+* To enable persistent storage of Prometheus time-series data, set this variable to `true` in the Ansible inventory file:
```

Comment:
I realize that the order doesn't matter, but for consistency's sake, I'd make this an OL instead of a UL.

Reply:
Made this into a one-step procedure, it's both consistent with our style and does not impose order (especially important because the user can skip any of the steps).